appcelerator/titanium_mobile
# fix(android): toolbar items do not update when changing dark/light theme

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28434

**Summary:**
- Child views added to `Ti.UI.Toolbar` via its `items` properties do not update themselves after changing dark/light theme.
- Only an issue when dynamically changing existing window.
- Not an issue if view are added via the toolbar's `add()` method.

**Test:**
1. Build and run the below JS code on Android 10 or higher.
2. Tap on the "Home" button after app launches. (Do NOT back out.)
3. Go to the Android system Settings -> Display screen.
4. Toggle the dark/light theme switch.
5. Resume the Titanium app.
6. Verify toolbar button colors have changed. (Light blue for dark mode. See screenshots [here](https://jira.appcelerator.org/browse/TIMOB-28434).)

```javascript
function createButton(name) {
	const button = Ti.UI.createButton({
		title: name,
		style: Ti.UI.BUTTON_STYLE_OPTION_NEUTRAL,
	});
	button.addEventListener("click", function() {
		alert(name + " was clicked on.");
	});
	return button;
}

const window = Ti.UI.createWindow();
const toolbar = Ti.UI.createToolbar({
	items: [
		createButton("Button 1"),
		createButton("Button 2"),
		createButton("Button 3")
	],
	elevation: 2,
	top: 0,
	width: Ti.UI.FILL,
});
window.add(toolbar);
window.open();
```
